README.md

```diff
@@ -94,66 +92,62 @@ params   = { object_purpose: 'PURCHASE',
 Let's take a quick look at what the `Shipment` object looks like:
 
 ```ruby
-require 'awesome_print'
+require 'awesome-print'
 ap @shipment
 # {
-#        "carrier_accounts" => [],
-#            "address_from" => "a704eada7494bb1be6184ef64b1646db",
-#              "address_to" => "92b43fbfa3641644beb32996042eb57a",
-#          "address_return" => "a1f64ba14b7e41b86a0446de4ebbd769",
-#                  "parcel" => "92df4baac73ea6131940c0d315d70a7d",
-#         "submission_date" => "2016-07-06T20:33:02.211Z",
-#               "return_of" => nil,
-#     "customs_declaration" => nil,
-#        "insurance_amount" => "0",
-#      "insurance_currency" => nil,
-#                   "extra" => {},
-#             "reference_1" => "",
-#             "reference_2" => "",
-#               "rates_url" => "https://api.goshippo.com/v1/shipments/a336daf87a8e442992a68daa6622758f/rates/",
-#                "messages" => [ ] # ommitted for brevity,
-#              "rates_list" => [ ] # ommitted for brevity.
+#   "carrier_accounts"    => [],
```

Comment:
yeah just the spacing seemed strange to me, and here's what changed apart from that:
- address fields are expanded instead of being ids
- submission_date --> shipment_date
- return_of --> is_return
- insurance and references inside extra
- delete rates_url
- rates_list --> rates